## 📋 Quality Gate Results

| Check | Status |
|-------|--------|
| Knip | ✅ success |
| Lint | ❌ failure |
| Build | ✅ success |
| Infra Tests | ❌ skipped |
| Unit Tests | ❌ skipped |
| Component Tests | ❌ skipped |
| Perf Tests | ❌ skipped |
| Visual Tests | ❌ skipped |


### ❌ Lint Failure Details
```text

> hrm@0.30.0 lint /home/runner/work/hrm/hrm
> eslint app/ components/ constants/ context/ hooks/ lib/ services/ tests/ types/ utils/ server.ts proxy.ts --cache


/home/runner/work/hrm/hrm/tests/unit/hooks/useSpotifyWebPlayback.test.ts
  127:14  error  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
  142:14  error  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any

✖ 2 problems (2 errors, 0 warnings)

 ELIFECYCLE  Command failed with exit code 1.
```
### ❌ Infrastructure Test Failure Details
```text
Log file not found.
```
### ❌ Unit Test Failure Details
```text
Log file not found.
```
### ❌ Component Test Failure Details
```text
Log file not found.
```
### ❌ Visual Test Failure Details
```text
Log file not found.
```
### ❌ Performance Test Failure Details
```text
Log file not found.
```
⚠️ **Some checks failed.** Full logs available in [workflow artifacts](https://github.com/arii/hrm/actions/runs/21698347000).

---
> Report generated for commit: `326f44536f9ef1349e979e6ea60e359860c8b9ed`

--- a/tests/unit/hooks/useSpotifyWebPlayback.test.ts
+++ b/tests/unit/hooks/useSpotifyWebPlayback.test.ts
@@ -124,7 +124,7 @@ describe('useSpotifyWebPlayback', () => {
 
     // Find the 'ready' listener
     const readyCall = mockPlayer.addListener.mock.calls.find(
-      (call: any[]) => call[0] === 'ready'
+      (call: unknown[]) => call[0] === 'ready'
     )
     expect(readyCall).toBeDefined()
     const readyCallback = readyCall[1]
@@ -139,7 +139,7 @@ describe('useSpotifyWebPlayback', () => {
 
     // Find the 'not_ready' listener
     const notReadyCall = mockPlayer.addListener.mock.calls.find(
-      (call: any[]) => call[0] === 'not_ready'
+      (call: unknown[]) => call[0] === 'not_ready'
     )
     expect(notReadyCall).toBeDefined()
     const notReadyCallback = notReadyCall[1]

